common/cache/domainCache.go

```diff
@@ -952,3 +955,10 @@ func (entry *DomainCacheEntry) IsSampledForLongerRetention(
 	}
 	return false
 }
+
+func (entry *DomainCacheEntry) HoursToAddToRetentionDuration() time.Duration {
```

Comment:
based on a method name, expected output is duration in hours, but no time.hour is used.
options: 
* change name to not mention hours
* add time.Hour